### PR TITLE
Consolidate scanner logic in global scanner package

### PR DESCRIPTION
**Pull request by lamaks**
#### Description of changes

Consolidate scanner logic in global scanner package

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Added relevant unit test for your changes. (`yarn test`)
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [ ] Ran precheckin (`yarn precheckin`)

### CODE REVIEW

File: packages/cli/src/scanner/ai-scanner.ts [hunk 1]
```diff
@@ -13,9 +11,7 @@ export class AIScanner {
     public async scan(url: string, chromePath?: string, sourcePath?: string): Promise<AxeScanResults> {
         try {
             console.log(`Starting accessibility scanning of URL ${url}.`);
-
             await this.page.create(chromePath);
-            await this.page.enableBypassCSP();
```
Comment (pownkel) [inline packages/cli/src/scanner/ai-scanner.ts:18]: Why do we no longer need this?

Comment (pownkel) [reply packages/cli/src/scanner/ai-scanner.ts:18]: Nevermind, it's because we do this anyway elsewhere in Page

Comment (lamaks) [reply packages/cli/src/scanner/ai-scanner.ts:18]: yes here https://github.com/microsoft/accessibility-insights-service/blob/d1fc305781244ad3b1819f2f45db02e01f6a9c80/packages/scanner-global-library/src/page-configurator.ts#L15

**Review by pownkel — APPROVED** [review]
LGTM

**Pull request merged**